staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher_test.go

```diff
@@ -424,29 +165,6 @@ func withDefaults(options *setupOptions) {
 	options.clock = clock.RealClock{}
 }
 
-func withClusterScopedKeyFunc(options *setupOptions) {
-	options.keyFunc = func(obj runtime.Object) (string, error) {
-		return storage.NoNamespaceKeyFunc(options.resourcePrefix, obj)
-	}
-}
-
-func withSpecNodeNameIndexerFuncs(options *setupOptions) {
-	options.indexerFuncs = map[string]storage.IndexerFunc{
-		"spec.nodeName": func(obj runtime.Object) string {
-			pod, ok := obj.(*example.Pod)
-			if !ok {
-				return ""
-			}
-			return pod.Spec.NodeName
-		},
-	}
-}
-
-func testSetup(t *testing.T, opts ...setupOption) (context.Context, *Cacher, tearDownFunc) {
-	ctx, cacher, _, tearDown := testSetupWithEtcdServer(t, opts...)
-	return ctx, cacher, tearDown
-}
-
 func testSetupWithEtcdServer(t *testing.T, opts ...setupOption) (context.Context, *Cacher, *etcd3testing.EtcdTestServer, tearDownFunc) {
```

Comment:
this method is now duplicated in the new package, it is hard to move to a new package since it references `cacher.Cacher` which would trigger an import cycle.